### ISSUE TITLE
SC 2.4.11/2.4.12 Clarify third bullet point

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#1291]
> Success Criterion 2.4.12 Focus Appearance (Enhanced)
> …
> * **Unobscured:** No part of the focus indicator is hidden by author-created content.

I think this point in the SC is already addressed by 2.4.11 Focus Appearance (Minimum). I don't think it is possible to say that a user interface component meets a AAA criterium and therefor does not need to meet the AA criterium that also applies to it. AAA criteria are additive.

For example, I would assume that the bullet point **Adjacent Contrast** from 2.4.11 needs to be met in order to conform to 2.4.12. Or, putting it the other way around, if you failed a AA criterion, you cannot get AAA conformance, so claiming meeting the AAA criterion is not useful on its own.

**Comment by alastc** [comment]
> I think this point in the SC is already addressed by 2.4.11 Focus Appearance (Minimum). 

I don't think that's the case, the 2.4.11 is:
> Unobscured: The item with focus is **not entirely** hidden by author-created content.

(Emphasis added).

2.4.11 requires it is not _entirely_ hidden, the 2.4.12 one is that it cannot be hidden at all, i.e. no overlap from headers/footers at all.

Surely that is similar to meeting a level of contrast at AA, and a higher level at AAA?

**Comment by yatil** [comment]
Ah, thanks for clarifying the emphasis here. Sometimes you look at these things and you just don’t see it. Maybe it would be clearer to say:

In 2.4.11: Not fully obscured
In 2.4.12: Not obscured

to make the difference more glanceable.

It think we need to define what “not entirely hidden” means. Is 1px OK, a 5pixel area? If I put focus outline around the focused item and then “obscure the item with focus” using a pseudo :before or :after element, do I still meet the SC? (I personally find that “not obscured at all” is a reasonable criteria for a AA criterion 😉)

**Comment by alastc** [comment]
Yea, it's tricky, worth having a look over where it came from:
- #952 about sticky footers/headers and 2.4.7.
- #1145 for adding this bullet on obscurement.
- #1271 about whether it turns into an exception.

We considered adding a measure for the obscurement, but it turns into another maths problem. It seemed better overall to indicate that it was not a good thing at AA, and have the AAA be the without-compromise version.

So for the clarification, do you mean something like this?
> Not fully obscured: The item with focus is not entirely hidden by author-created content.

**Comment by yatil** [comment]
> So for the clarification, do you mean something like this?
>
>> Not fully obscured: The item with focus is not entirely hidden by author-created content.

Yes, just to make the difference more glanceable. 

I wonder if we – to keep the maths the same – could say “not less than the focus indication area is obscured”, which would come to this:

> Not fully obscured: The item with focus has a visible area that is at least as big as the focus indication area.

**Comment by juleskuehn** [comment]
> Not fully obscured: The item with focus has a visible area that is at least as big as the focus indication area.

The focus indicator itself also needs to be "not fully obscured" for it to be obvious that focus has moved to that item. Perhaps something like this:
> Not fully obscured: The item with focus has a visible focus indicator area equal to or greater than 1/4 of the focus indication area.

**Comment by alastc** [comment]
@juleskuehn we already have metrics for the focus indicator, the reason for this bullet is to add something for overlapping content.

This appears circular to me, as the focus-indication area is defined as what is visible:
>  The item with focus has a visible focus indicator area equal to or greater than 1/4 of the focus indication area.

Bullets 1-3 are about the focus indicator of the element, then bullet 4 is about whether the whole element is obscured.

Unless someone can point out a very simple metric, it seems best to have a general 'not fully obscured', and 'not obscured' as Eric suggested.

I've created a branch & PR for that, #1322

**Comment by juleskuehn** [comment]
Sorry if that was less-than-clear. What I am trying to address is why Focus Appearance (Enhanced) mentions the focus **indicator** not being obscured at all:

> No part of the focus indicator is hidden by author-created content.

But Focus Appearance (Minimum) does not mention the focus indicator being obscured, but rather the focused **item** being obscured:

> The item with focus is not entirely hidden by author-created content.

For Focus Appearance (Minimum) it would be better, and more consistent, to state:

> The focus indicator is not entirely hidden by author-created content.
